src/RetryTrait.php

```diff
@@ -116,14 +115,7 @@ private function executeRetryDelayFunction($retryAttempt)
             array_unshift($delayMethodArgs, $retryAttempt);
             call_user_func_array([$this, $delayMethod], $delayMethodArgs);
         }
-    }
 
-    private function exponentialBackoff($retryAttempt, $maxDelaySeconds = 60)
```

Comment:
that method was unused

Reply:
It's included as a way to have a build-in exponential backoff, and is mentioned in the `README`:

```php
/**
 * @retryDelayFunction exponentialBackoff
 */
```

Please add it back!